addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10671,7 +10671,25 @@ msgctxt "#19296"
 msgid "No PVR add-on enabled"
 msgstr ""
 
-#empty strings from id 19297 to 19498
+#. Label for filtered recordings view value: 'show all'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19297"
+msgid "All recordings"
+msgstr ""
+
+#. Label for filtered recordings view value: 'show only unwatched'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19298"
+msgid "Unwatched"
+msgstr ""
+
+#. Label for filtered recordings view value: 'show only watched'
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19299"
+msgid "Watched"
```

Comment:
Yep, but the implementation (I just reused) relies on three strings with numeric values in a row, and because one of the strings for videos ("All videos") did not fit, I needed to reinvent three new strings. Not a problem, if you ask me. The context is another, this it even cn be dangerous to reuse the strings, we had that problem already..